frontend/src/util/tinymceInit.js

```diff
@@ -66,6 +66,16 @@ import { uniqueId } from 'lodash'
       }
     }
 
+    const hiddenTinymceFileInput = document.createElement('input')
+    hiddenTinymceFileInput.id = 'hidden_tinymce_fileinput'
```

Comment:
id is actually not necessary anymore, but we can leave it for clarity when inspecting the page with the developer tools